packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -324,41 +305,21 @@ contract SphinxUtils is SphinxConstants {
             );
         }
 
-        assertValidOwners(_config);
-
-        require(
-            _config.threshold > 0,
-            "Sphinx: You must set your 'sphinxConfig.threshold' to a value greater than 0 before calling this function."
-        );
-        require(
-            _config.owners.length >= _config.threshold,
-            "Sphinx: Your 'sphinxConfig.threshold' field must be less than or equal to the number of owners in your 'owners' array."
-        );
         require(
             bytes(_config.projectName).length > 0,
-            "Sphinx: Your 'sphinxConfig.projectName' cannot be an empty string. Please enter a project name."
-        );
-
-        address[] memory duplicateOwners = getDuplicatedElements(_config.owners);
-        require(
-            duplicateOwners.length == 0,
-            string(
-                abi.encodePacked(
-                    "Sphinx: Your 'sphinxConfig.owners' array contains duplicate addresses: ",
-                    toString(duplicateOwners)
-                )
-            )
```

Comment:
We no longer need these checks since we do all of this validation when the project is created via the website. See the corresponding PR in the website repo for all of that logic. I've also included a comprehensive suite of tests there.